docs/configurations.md

```diff
@@ -101,164 +127,81 @@ The following describes the configurations available for each storage.
+## Other ScalarDB configurations
 
-Other configurations are available for ScalarDB:
+The following are additional configurations available for ScalarDB:
 
 | Name                                                             | Description                                                                                                                                                                                                       | Default              |
 |------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------|
 | `scalar.db.metadata.cache_expiration_time_secs`                  | ScalarDB has a metadata cache to reduce the number of requests to the database. This setting specifies the expiration time of the cache in seconds.                                                               | `-1` (no expiration) |
 | `scalar.db.active_transaction_management.expiration_time_millis` | ScalarDB maintains ongoing transactions, which can be resumed by using a transaction ID. This setting specifies the expiration time of this transaction management feature in milliseconds.                       | `-1` (no expiration) |
-| `scalar.db.default_namespace_name`                               | The given namespace name will be used by operations that do not already specify a namespace. If you would like to use this setting with ScalarDB Server, configure this setting on the client-side configuration. |                      |
+| `scalar.db.default_namespace_name`                               | The given namespace name will be used by operations that do not already specify a namespace. |                      |
 
-## Configuration examples
+### Two-phase commit support
 
-This section shows several configuration examples.
+ScalarDB supports transactions with a two-phase commit interface. With transactions with a two-phase commit interface, you can execute a transaction that spans multiple processes or applications, like in a microservice architecture.
 
-### Example 1
+For details about using two-phase commit, see [Transactions with a Two-Phase Commit Interface](two-phase-commit-transactions.md).
 
-```
-[App (ScalarDB Library with Consensus Commit)] ---> [Underlying storage/database]
-```
-
-In this configuration, the app (ScalarDB Library with Consensus Commit) connects to the underlying storage/database (in this case, Cassandra) directly.
-Note that this configuration exists only for development purposes and is not recommended for production use.
-This is because the app needs to implement the [scalar-admin](https://github.com/scalar-labs/scalar-admin) interface to take transactionally consistent backups for ScalarDB, which requires an extra burden for users.
+## Configuration example
```

Comment:
@brfrn169 In version [`3.8`](https://scalardb.scalar-labs.com/docs/3.8/configurations/#configuration-example-2---app-scalardb-server-and-database) and earlier of this doc, we have a second example that shows `App --> ScalarDB Server --> Underlying storage or database`, which I mistakenly removed for the `master`, `3.10`, and `3.9` version of this doc.

Is it OK if we include ScalarDB Cluster at the bottom of this section as follows?

> **Note**
> 
> The following suggestion is almost identical to the [second example in the `3.8` version of this doc](https://github.com/scalar-labs/scalardb/blob/b449ae77527b51bb2d56b8eed58a588308b93b0f/docs/configurations.md?plain=1#L226C26-L226C26).

````suggestion
### Configuration example #2 - App, ScalarDB Cluster, and database

```mermaid
flowchart LR
    app["App -<br />ScalarDB library with gRPC"]
    cluster["ScalarDB Cluster -<br />(ScalarDB library with<br />Consensus Commit)"]
    db[(Underlying storage or database)]
    app --> cluster --> db
```

In this example configuration, the app (ScalarDB library with gRPC) connects to an underlying storage or database (in this case, Cassandra) through ScalarDB Cluster.

{% capture notice--info %}
**Note**

This configuration is acceptable for production use because ScalarDB Cluster implements the [Scalar Admin](https://github.com/scalar-labs/scalar-admin) interface, which enables you to take transactionally consistent backups for ScalarDB by pausing ScalarDB Cluster.

{% endcapture %}

<div class="notice--info">{{ notice--info | markdownify }}</div>

The following is an example of the configuration for connecting the app to the underlying database through ScalarDB Cluster:

```properties
# Transaction manager implementation.
scalar.db.transaction_manager=cluster

# ScalarDB Cluster host.
scalar.db.contact_points=<SCALARDB_CLUSTER_HOST>

# ScalarDB Cluster port.
scalar.db.contact_port=<SCALARDB_CLUSTER_PORT>

# Storage implementation.
scalar.db.storage=cassandra

# Comma-separated contact points.
scalar.db.contact_points=<CASSANDRA_HOST>

# Credential information to access the database.
scalar.db.username=<USERNAME>
scalar.db.password=<PASSWORD>
```
````

Reply:
Yes, I think we can add a ScalarDB Cluster example here. Also, we should also mention ScalarDB Cluster is a component we provide only in the Enterprise edition here again.

Reply:
Thank you for confirming! Also, I've revised the above suggestion to mention `... ScalarDB Cluster, which is a component that is available only in the ScalarDB Enterprise edition.`

I'll add this example to this doc on `3.10` and `3.9` as well.